www/lendingbot.html

```diff
@@ -62,12 +62,12 @@
     <nav class="navbar navbar-default navbar-fixed-top">
         <div class="container">
             <div class="row">
-                <div class="col-xs-6">
+                <div class="col-xs-8">
                     <h4>
                         Poloniex Lending Bot
                     </h4>
                 </div>
-                <div class="col-xs-6" style="text-align:right;margin: 10px auto;" >
+                <div class="col-xs-4" style="text-align:right;margin: 10px auto;" >
```

Comment:
My tests show that this forces the date & time to use 2 lines instead of 1, not sure why is this an improvement.